docs/versioning.md

```diff
@@ -59,5 +59,6 @@ To see the list of available major versions of Powertools for AWS Lambda and whe
 
 | SDK                              | Major version | Current Phase        | General Availability Date | Notes                                                                                                                                                                |
 | -------------------------------- | ------------- | -------------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Powertools for AWS Lambda Python | 2.x           | General Availability | 10/24/2022                | See [Release Notes](https://github.com/aws-powertools/powertools-lambda-python/releases/tag/v2.0.0)                                                                  |
-| Powertools for AWS Lambda Python | 1.x           | End of Support       | 06/18/2020                | See [RFC](https://github.com/aws-powertools/powertools-lambda-python/issues/1459) and [upgrade guide](https://docs.powertools.aws.dev/lambda/python/latest/upgrade/) |
+| Powertools for AWS Lambda (Python) | 3.x           | Developer Preview    |                           | See [RFC](https://github.com/aws-powertools/powertools-lambda-python/issues/4189)                                                                  |
```

Comment:
we need to let customers know how to build/test this developer preview in the RFC later, plz.

Reply:
Ok ok, noted!